src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationConstants.cs

```diff
@@ -30,5 +30,110 @@ public static class Claims
             public const string Specialties = "urn:linkedin:specialties";
             public const string Summary = "urn:linkedin:summary";
         }
+
+        public const string EmailAddressField = "email-address";
+
+        // https://developer.linkedin.com/docs/fields/basic-profile
+        public static class ProfileFields
+        {
+            /// <summary>
+            /// A unique identifying value for the member.
+            ///This value is linked to your specific application.Any attempts to use it with a different application will result in a "404 - Invalid member id" error.
+            /// </summary>
+            public const string Id = "id";
```

Comment:
Could you add new lines between each new constant please? It makes the code easier to read and review.

Reply:
Done
Could you set up `.editorconfig` with this rule?

Reply:
Sure, that's something I can look at doing some time this week or next.

Reply:
#296